### ISSUE TITLE
Github packages

### ISSUE THREAD

**Issue by jonathan-irvin** [verdaccio/verdaccio#1789]
Forgive me if this is a duplicate, but I'm using verdaccio via docker in an offline cache (for a network isolated from the public internet).

There are some packages that I'm trying to cache that reference github forks with commitish references and I was wondering how that would work with verdaccio (given the clients wouldn't have access to github and only the verdaccio server)?

I would love some guidance on how best to approach this.

tl;dr 

How do I handle npm packages that reference github forks with verdaccio when my server nor clients would have access to github in a private network?

**Comment by juanpicado** [comment]
hi @jonathan-irvin 

Unfortunately, Verdaccio does not cache git repositories used as dependencies. Actually the node client package manager seems to download them directly, so, it cannot be possible with Verdaccio anyway.

I've heard a few times legacy projects still uses `git://` as repositories, but, I do not recommend following this practice and rather use semantic versioning and take advance of updates.

If you cannot change for some reason, then you might clone the git repositories locally and change the URL to file system base, eg: https://stackoverflow.com/questions/2519933/git-clone-repo-across-local-file-system-in-windows

I hope that helps.
